sal_interfaces/SALSubsystems.xml

```diff
@@ -441,19 +441,19 @@
 
 <SALSubsystem>
   <Name>ATCamera</Name>
-  <Description></Description>
+  <Description>Auxilliary telescope camera</Description>
   <IndexEnumeration>no</IndexEnumeration>
   <Generics>yes</Generics>
-  <ActiveDevelopers>Tony Johnson</ActiveDevelopers>
-  <Github>Yes, but private</Github>
-  <JenkinsTestResults>Not Available</JenkinsTestResults>
-  <RubinObsContact>Michael Reuter</RubinObsContact>
-  <CSCDocs>NA</CSCDocs>
+  <ActiveDevelopers>CCS Team</ActiveDevelopers>
+  <Github>https://github.com/lsst-camera-ccs/</Github>
+  <JenkinsTestResults>https://srs.slac.stanford.edu/releaseManagement/</JenkinsTestResults>
+  <RubinObsContact>Tony Johnson</RubinObsContact>
+  <CSCDocs>https://confluence.slac.stanford.edu/display/LSSTCAM/CCS+Subsystem+Manuals</CSCDocs>
   <ProductOwner>Brian Stalder</ProductOwner>
-  <RelatedDocuments>Yes, Michael knows</RelatedDocuments>
+  <RelatedDocuments>LSE-71</RelatedDocuments>
   <SoftwareLanguage>Java</SoftwareLanguage>
-  <RuntimeLanguages>IDL</RuntimeLanguages>
-  <VendorContact>Camera</VendorContact>
+  <RuntimeLanguages>IDL,Java</RuntimeLanguages>
+  <VendorContact></VendorContact>
   <Simulator>No</Simulator>
```

Comment:
This should be "Not Provided" or "Not Required"

Reply:
VendorContact should be "Not Applicable" since the camera software development is part of the Rubin Observatory project. In spite of the Camera team being funded by a different agency we do not consider them vendors.

Reply:
Fixed